source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidTurnBasedMultiplayerClient.cs

```diff
@@ -332,6 +332,14 @@ public void RegisterMatchDelegate(MatchDelegate del)
             }
         }
 
+        protected Action<TurnBasedMatch, bool> MatchDelegate
```

Comment:
Converted from public -> protected as asked in ->
https://github.com/playgameservices/play-games-plugin-for-unity/pull/2577/files/bd5239864ec45c5855e266415ec22ee8f72cc45e..f1ede0601f94754e3314e8d717368b07431fcdf2#r294236580